release/release.go

```diff
@@ -15,6 +15,61 @@ import (
 	"github.com/mongodb/mongo-tools/release/platform"
 )
 
+// The msi msiUpgradeCode must be updated when the minor version changes.
+var msiUpgradeCode string = "56c0fda6-289a-4fd0-a539-6711864146ba"
+
+// These are the binaries that are part of mongo-tools, relative
+// to the location of this go file.
+var binariesPath string = filepath.Join("..", "bin")
```

Comment:
Unfortunately, we can't do absolute paths without passing in the evergreen directory to the script, which I'd rather avoid doing. Evergreen uses uuids for directory names. By making them relative to the location of the go file, it makes the paths pretty safe.

Reply:
I don't think it's so bad now that I'm making these not global to the file.

Reply:
I think you can just use `filepath.Abs()` to make them absolute

Reply:
They are still being constructed using relative paths. We would be constructing a relative path then turning it into an absolute path. How would that be useful? Are you thinking for error messages? The `buildZip` and `buildTarball` functions are using relative paths, so we'd want to change those for consistency, as well.

Reply:
The primary reason I had in mind is that we are changing directories in the script, so relative paths may break if we exec things from a different dir (this is the same reason why we prefix all paths in the sqlproxy bash scripts with `${PROJECT_DIR}`, `${ARTIFACTS_DIR}`, etc).

Making the paths absolute for zip and tarball creation never occurred to me when making zips and tarballs because those are just using the project root dir, but I agree that those would be better as abs paths as well!